libs/coin-framework/src/operation.ts

```diff
@@ -210,3 +210,17 @@ export const isConfirmedOperation = (
   operation.blockHeight
     ? account.blockHeight - operation.blockHeight + 1 >= confirmationsNb
     : false;
+
+export const isAddressPoisoningOperation = (
+  operation: Operation,
+  account: AccountLike
+): boolean => {
+  const familiesImpacted = ["ethereum", "evm", "tron"];
```

Comment:
I don't consider it a blocker for this fix, but this isn't going in the right direction regarding coin modularization.

Potentially proper pattern could be to add an `isAddressPoisoningOperation` to the `AccountBridge`.
I'm reluctant to "pollute" the bridges, but at the same time anything that's isn't relevant / identical for _all_ coins should go through them.

Reply:
Yup agreed, and I also have mixed feelings about polluting the bridge with too many things... We also had this discussion regarding NFTs related features and I'm still disturbed a year later about having some methods being 100% specific to only 1 familly... 
Might be a discussion to have with Stephane so we can figure out a "good looking" way to maybe provide a second layer of functionnalities that are not "basic" stuff like this one ? 🤔

Reply:
One change that I can already do, is to move that array as an env ?

Reply:
:+1: we should have a discussion at some point with gre & Stephane about a proper strategy of when something should/shouldn't be added to bridges.

:+1: for the env too, I mean it's not a general solution but here it works because the behavior is exactly the same for all.